### PR TITLE
feature - improved profile ui

### PR DESCRIPTION
**Pull request by at-lytruong**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Comment by at-hieutran** [comment]
Fix conflict

File: MyApp/View/Controllers/Profile/ProfileTableViewCell.swift [hunk 1]
```diff
@@ -7,10 +7,16 @@
 //
 
 import UIKit
+import SwiftUtils
 
 class ProfileTableViewCell: UITableViewCell {
     @IBOutlet weak var iconImage: UIImageView!
     @IBOutlet weak var nameIconLabel: UILabel!
+    @IBOutlet weak var viewImage: UIView!
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Profile/ProfileTableViewCell.swift]: Naming

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Comment by at-hieutran** [comment]
LGTM

**Pull request merged**